lua/nvim-tree/view.lua

```diff
@@ -211,8 +211,12 @@ local function grow()
   local lines = vim.api.nvim_buf_get_lines(M.get_bufnr(), starts_at, -1, false)
   local max_length = M.View.initial_width
   for _, l in pairs(lines) do
-    if max_length < #l then
-      max_length = #l
+    -- count of unicode codepoints.
+    -- (#l counts the bytes in the string which is not desirable)
+    local _, count = string.gsub(l, "[^\128-\193]", "")
+    count = count + 3 -- add padding
+    if max_length < count then
+      max_length = count
```

Comment:
could you use the `vim.fn.strchars` instead ?

Reply:
Wasn't aware this function existed, I just amended my commit.